### ISSUE TITLE
Does not rate limit on root path "/"

### ISSUE THREAD

**Issue by anderslatif** [nfriedly/express-rate-limit#225]
I don't know if this is considered a "feature" but I've just discovered that there's no rate limiting - when you set it up as middleware for all routes - on "/". That's defeats the idea that someone might be calling your API too many times and you don't even want to call the callback in Express to serve them the content.

**Comment by nfriedly** [comment]
I think you may have configured something. Rate-limiting on `/` is a commonly used and well-tested feature, e.g.

https://github.com/nfriedly/express-rate-limit/blob/1f11f793d39780fe75d13620f739d200a94b3d2e/test/express-rate-limit-test.js#L267-L277

Can you post your code?

**Comment by anderslatif** [comment]
I figured it out. The crazy thing is that when I serve a folder statically with Express then even if there is not a single route defined it will fallback to index.html when "/" is requested. 

That means that if I create a route on "/" that serves a file then it won't be called by Express. Thus it's not a bug in the rate-limiter cause it's not even called. 

I hope my explanation makes sense. Closing this issue.
